<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifySwitchDialog</class>
 <widget class="QDialog" name="ModifySwitchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>138</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>修改交换机信息</string>
  </property>
  <widget class="QLineEdit" name="switchYLineEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="staticSwitchNameLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>名字：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="switchNameLineEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="staticSwitchYLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Y坐标(km)：</string>
   </property>
  </widget>
  <widget class="QLabel" name="staticSwitchXLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>X坐标(km)：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="modifySwitchButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="switchXLineEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
